index.html

```diff
@@ -331,18 +609,15 @@ <h2>
           <dfn>exitPointerLock()</dfn> method
         </dt>
         <dd>
-          <p>
-            Initiates an exit from pointer lock state if currently locked to a
-            target whose <a>shadow-including root</a> is this document, and
-            sends a <a>pointerlockchange</a> event when the lock state has been
-            exited.
-          </p>
-          <p>
-            The system mouse cursor must be displayed again and positioned at
-            the same location that it was when pointer lock was entered (the
-            same location that is reported in <code>screenX</code>,
-            <code>screenY</code>, when the pointer is locked).
-          </p>
+          <ol>
+            <li>
```

Comment:
I think you need a step for what happens if people call exitPointerLock() when you're currently not in a pointer lock. Something like

> 1. If the user agent's pointer-lock target is null, then return.

Reply:
Added.